lms/djangoapps/courseware/tests/test_date_summary.py

```diff
@@ -114,7 +112,7 @@ def assert_block_types(self, course, user, expected_blocks):
          {'verification_status': 'expired'},
          (TodaysDate, CourseEndDate, VerificationDeadlineDate)),
         # Verified enrollment with `approved` photo-verification during course run
-        ({'days_till_start': -10, },
+        ({'days_till_start': -10},
```

Comment:
what happened here?

Reply:
Lol. just an oddity I spotted in the file. I think it used to contain multiple arguments and now it doesn't, but they just left in the `,`. This is a purely style change. No functional difference.